packages/client-direct/src/README.md

```diff
@@ -43,3 +43,42 @@ curl -X GET "http://localhost:3000/fine-tune/8566c47a-ada8-441c-95bc-7bb07656c4c
   -H "Content-Type: application/json" \
   -H "Authorization: Bearer jvBpxrTNqGqhnfQhSEqCdsG6aTSP8IBL".
 ```
+
+
+## Verifiable Attestations
```

Comment:
Love this! 

I think the easiest way to insert into documentation is to add this as a section called **Enable Verifiable Logs** for the **Eliza in TEE** Doc `docs/docs/advanced/eliza-in-tee.md`. Then add a reference to your discord or discord user to contact about the implementation.

Reply:
I think we can work on this together for formatting & further enhancements/tutorials down the road. Separate into to two Plugin Sections:
- TEE Plugin
   - Core Components
- TEE Verifiable Log Plugin
   - Core Components
- Tutorial
   - Enable Verifiable Log
- Conclusion
   - Mention contributors for implementation and who to reach out to learn more about Verifiable Logs in TEE

Reply:
No problem, let's work on this together. that’s exactly the idea behind building this plugin.

Let me explain further:  
- #1259 : This is a standalone log plugin that doesn’t modify other plugins or the client.  
- #1260 : This modifies the direct client to provide a query interface for verifiable logs.

@HashWarlock @odilitime